examples/basic/main.tf

```diff
@@ -64,6 +64,16 @@ resource "aws_security_group_rule" "alb_ingress_80" {
   ipv6_cidr_blocks  = ["::/0"]
 }
 
+resource "aws_efs_file_system" "efs" {
+  creation_token = "blah"
```

Comment:
This needs to be unique to make the resource idempotently.
It is also created by Terraform by default. Why do you explicitly set it to a fixed value?

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system#creation_token

https://docs.aws.amazon.com/efs/latest/ug/API_CreateFileSystem.html

Reply:
Hello,
It's just a copy/paste, I will remove it

Reply:
Done